src/ResourceManager/Websites/Commands.Websites.Test/ScenarioTests/WebsitesTests.ps1

```diff
@@ -0,0 +1,77 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Tests creating a new website.
+#>
+function Test-CreatesNewSimpleWebsite
+{
+	# Setup
+	$rgname = "Default-Web-WestUS"
+	$wname = "ngoliPSWebsite"
+	$location = "West US"
+	$webHostingPlan = "NGoliStandard"
+	$apiversion = "2014-04-01"
+	$resourceType = "Microsoft.Web/sites"
+
+
+	try
+	{
+			# Test
+			$actual = New-AzureWebsite -ResourceGroupName $rgname -WebsiteName $wname -Location $location -WebHostingPlan $webHostingPlan 
```

Comment:
No it is expected that the users provide the existing webshosting plan name otherwise it will assert